### PR TITLE
add moreores support

### PR DESCRIPTION
**Pull request by tenplus1**
added moreores tool support

### CODE REVIEW

File: init.lua [hunk 1]
```diff
@@ -187,3 +187,36 @@ minetest.override_item("default:shovel_mese", {
   description = toolranks.create_description("Mese Shovel", 0, 1),
   after_use = toolranks.new_afteruse,
 })
+
+if minetest.get_modpath("moreores") then
+
+minetest.override_item("moreores:pick_mithril", {
```
Comment (lisacvuk) [inline init.lua]: Can you please indent this block?

File: init.lua [hunk 2]
```diff
@@ -66,7 +67,7 @@ function toolranks.new_afteruse(itemstack, user, node, digparams)
       gain = 2.0,
     })
   end
-  level = toolranks.get_level(dugnodes)
+  local level = toolranks.get_level(dugnodes)
```
Comment (lisacvuk) [inline init.lua:70]: y that looks better

File: init.lua [hunk 3]
```diff
@@ -58,6 +58,7 @@ function toolranks.new_afteruse(itemstack, user, node, digparams)
   -- Only count nodes that spend the tool
   if(digparams.wear > 0) then
    dugnodes = dugnodes + 1
+   itemmeta:set_string("dug", dugnodes)
```
Comment (lisacvuk) [inline init.lua:61]: Aaand that also makes more sense.

File: init.lua [hunk 4]
```diff
@@ -187,3 +187,36 @@ minetest.override_item("default:shovel_mese", {
   description = toolranks.create_description("Mese Shovel", 0, 1),
   after_use = toolranks.new_afteruse,
 })
+
+if minetest.get_modpath("moreores") then
+
```
Comment (lisacvuk) [inline init.lua]: maybe delete this empty line? I'm not sure if that would look better.

File: init.lua (hunk 1 above)
Comment (lisacvuk) [reply init.lua]: Also, this whole part seems to be using tabs instead of spaces.

**Review by lisacvuk — CHANGES_REQUESTED** [review]
While I was more considering other mods add support for this one, I doubt moreores will actually do it, so this PR is a nice idea, thanks :D

File: init.lua (hunk 1 above)
Comment (tenplus1) [reply init.lua]: ehehe, tabs be the norm but I'll change to spaces :)

**Pull request merged**